pages/partials/questionFooter.ejs

```diff
@@ -90,34 +90,26 @@
       </div>
       <% } %>
     </form>
-
-    <script>
-      $('form').on('submit', function(e) {
-          if (!$(this).data('submitted')) {
-              $(this).data('submitted', true);
-
-              var form = $(this);
-              form.find('.disable-on-submit').each(function (index) {
-                  // Create a disabled clone of the submit button
-                  $(this).clone(false).removeAttr('id').prop('disabled', true).insertBefore($(this));
-
-                  // Hide the actual submit button and move it to the beginning of the form
-                  $(this).hide();
-                  form.prepend($(this));
-              });
-
-          } else {
-              e.preventDefault();
-          }
-      });
-
-      $('a.disable-on-click').on('click', function() {
-          $(this).addClass('disabled');
-      });
-    </script>
   </div>
 <% } %>
+<script>
+    $('form.question-form').on('submit', function() {
+        if (!$(this).data('submitted')) {
+            $(this).data('submitted', true);
+
+            $(this).find('.disable-on-submit').each(function() {
+                // Create a disabled clone of the submit button
+                $(this).clone(false).removeAttr('id').prop('disabled', true).insertBefore($(this));
```

Comment:
Maybe a dumb question, but what's the need for creating a new disabled button instead of setting the existing one to be disabled?  I see that you're doing that for link buttons so not sure what the difference is.

Reply:
Ah that reminds me, I should update the comments. It took me a while to figure out why.

Long story short, disabled buttons don't POST (all browsers, not just Edge). Disabling the links is fine since they don't need to POST anything, but we need the buttons to POST the `__action`.

After trying a bunch of other hacks, Matt's original cloning method seems to be the most cross-compatible, progressive-enhancing workaround. The fix was just removing the `else` clause.